testing/grpcwebclient/browserscript.ts

```diff
@@ -53,6 +46,17 @@ async function runTestCase(data: number[]): Promise<number[]> {
   return Array.from(result.serializeBinary());
 }
 
+function log(msg: string) {
+  let now = new Date();
+  // eek, there must be a better way...
```

Comment:
Try:

`let timestamp = now.toLocaleTimeString("en-US", { hour12: false })`

Reply:
That doesn't include millisecond precision, which is what I was going for.

This function is actually unused currently, but I had added temporary log lines to troubleshoot something, and I needed millisecond precision to really understand the timing of some of the event callbacks.

Reply:
I decided to just remove the function since it's not actually used right now. (Seemed like too much work to add more log lines and iterate on it until it looked good with a new formulation... We can apply something like this if/when we need it in the future.)